generators/Generators.fs

```diff
@@ -170,6 +170,14 @@ type Pangram() =
 type PigLatin() =
     inherit Exercise()
 
+type PrimeFactors() =
+    inherit Exercise()
+    
+    override this.RenderInput (canonicalDataCase, key, value) =
+        base.RenderInput (canonicalDataCase, key, value) |> sprintf "%sL"
+    
+    override this.RenderSutProperty canonicalDataCase = "primeFactorsFor"
```

Comment:
I would rather rename the property in our example and stub file. Could you remove this line and adjust the example and stub implementation to follow the new name?

Reply:
I will remove this line and adjust the stub implementation and examples.

Regarding the example implementation, I think you are right. As you said I tried building and running the implementation. Apparently it was running for a while and memory was shooting up.
Should we go change that too ?

Reply:
@vrnithinkumar If you would, yes please! It is important that our example implementations run in a reasonable time frame, as that directly influences the build server time.